account_page.php

```diff
@@ -257,18 +269,26 @@
 		</fieldset>
 	</form>
 </div>
-<?php # check if users can't delete their own accounts
-if( ON == config_get( 'allow_account_delete' ) ) { ?>
 
-<!-- Delete Button -->
 <div class="form-container">
+	<fieldset>
+	<span class="submit-button">
+<!-- Revoke Token: not we know that user account is not protected -->
+<form method="post" action="account_revoke_token.php">
```

Comment:
Yes.  But the whole form is only viable because we have ensure that user in unprotected, hence, I removed the comment.